src/pages/sign-up/Email/index.tsx

```diff
@@ -46,8 +54,14 @@ export const Email = () => {
         <div className="form-container">
           {!referrerUser && (
             <IntroHeader
-              title="Create an account"
-              description={type === 'user' ? 'Sign up and start making an impact' : 'Sign up to hire professional'}
+              title={isTechSummit ? 'Tech for Impact Summit 2024' : 'Create an account'}
```

Comment:
We may check the condition in this way: if eventName is not empty, title would be the eventName, otherwise `create an account`